docroot/modules/custom/sitenow_migrate/src/EventSubscriber/MigratePostImportEvent.php

```diff
@@ -82,22 +82,6 @@ public function __construct(EntityTypeManager $entityTypeManager, LoggerInterfac
     $this->entityTypeManager = $entityTypeManager;
     $this->logger = $logger;
     $this->connection = $connection;
-
-    // Switch to the D7 database.
```

Comment:
@cory-skeers I'm trying to config-ignore the sitenow_migrate group so that sites can set DB connection info other than to DD. That change seems to have broken this code for some reason. I moved it to the d7_page case statement below for the time being. I'm not sure why it can't find the database key anymore.

I think we'll need to test out a "standard" site migration with this PR. Would you be able to head that up?

Reply:
I'll take a look!

Reply:
I think this was caused by the migrate_drupal `migration_plugins_alter` hook doing some wonky stuff. It also caused that DB error mentioned in the thread. I unset it in 206d9e0 and this code seems to be working now in the constructor.